sdk/cosmos/azure-cosmos/azure/cosmos/_base.py

```diff
@@ -26,28 +26,29 @@
 import datetime
 import json
 import uuid
-import urllib
 import binascii
 
 from . import auth
 from . import documents
 from . import partition_key
 from . import http_constants
-from . import _constants as constants
 from . import _runtime_constants
 
 import six
 from six.moves.urllib.parse import quote as urllib_quote
 from six.moves import xrange
 
-def GetHeaders(cosmos_client_connection,
-               default_headers,
-               verb,
-               path,
-               resource_id,
-               resource_type,
-               options,
-               partition_key_range_id = None):
+
+def GetHeaders(
```

Comment:
This PR is only for things that PyLint catches (and only some of them -- I think we should merge this and do the rest in a follow on to avoid risking conflicts)